src/_common/community/community.model.ts

```diff
@@ -70,7 +70,7 @@ export class Community extends Collaboratable(Model) {
 	thumbnail?: MediaItem;
 	header?: MediaItem;
 	theme!: Theme | null;
-	game!: Game | null;
+	games!: Game[] | null;
```

Comment:
Add comment explaining this only has the visible games.

Reply:
Actually, I think this'll be better if for collaborators it'd show all the games (and we can put a label or a tick on the hidden ones saying "only you can see this" or something) and for the rest of the users only the visible games would be shown. This actually simplifies handling the games list in other components too.